src/com/perl5/lang/perl/util/PerlRunUtil.java

```diff
@@ -83,6 +100,82 @@ public static GeneralCommandLine getPerlCommandLine(@NotNull Project project, @N
 		return commandLine;
 	}
 
+	private static List<String> getPerlLibraries(@NotNull Project project, PerlConfiguration runProfile, @Nullable VirtualFile scriptFile)
+	{
+		if (PlatformUtils.isIntelliJ())
+		{
+			// try to get used SDK
+			Sdk sdk;
+			if (runProfile != null && runProfile.isUseAlternativeSdk())
+			{
+				sdk = ProjectJdkTable.getInstance().findJdk(runProfile.getAlternativeSdkPath());
+			}
+			else
+			{
+				Module moduleForFile = scriptFile == null ? null : ModuleUtilCore.findModuleForFile(scriptFile, project);
+
+				sdk = getModuleSdk(moduleForFile);
+				if (sdk == null)
+				{
+					// found in project
+					sdk = ProjectRootManager.getInstance(project).getProjectSdk();
+					if (sdk == null || sdk.getSdkType() != PerlSdkType.getInstance())
+					{
+						// looking for any perl module in project
+						for (Module module : ModuleManager.getInstance(project).getModules())
```

Comment:
"Try anything" approach is not good and may be hard to maintain in future. We should suggest that project is properly configured.